### ISSUE TITLE
Intermittent test failure

### ISSUE THREAD

**Issue by whosgonna** [whosgonna/Zabbix-Tiny.pm#19]
The following failure is intermittently observed in test 02_apiinfo.version.t:

```
#   Failed test 'Still able to retrieve version after login. (No auth parameter sent)'
#   at t/02_apiinfo.version.t line 106.
# died: malformed number (leading zero must not be followed by another digit), at character offset 40 (before "492714094}") at /usr/home/ben/Zabbix-Tiny/.build/77f9lV9s/blib/lib/Zabbix/Tiny.pm line 141.
```

Will look into this before pushing to CPAN.

**Comment by whosgonna** [comment]
Quick guess:
The id is set as a random string of numbers, thus on one out of ten attempts, the leading digit should be a zero.  It looks like the problem is the JSON module is treating this as a number, rather than a string, which is failing to encode when the leading number is a zero.

I had originally mis-read the JSON-RPC specification, and thought that the ID was **supposed** to be random.  On a re-read, this appears to be incorrect, and they should be sequential.  Currently, I'm re-generating the id with each ```prepare```, so the id should be a tracked property.  I think a hotfix is needed for the current encoding problem, and rectifying the id to a non-random number will need to be a separate (slightly larger) fix.

**Comment by whosgonna** [comment]
Fixed in 387859ce.  ID now increments sequentially.
